File: HorseRacing.Contracts/Models/Game/Dtos/GameDto.cs
using HorseRacing.Domain.GameAggregate.Enums;

namespace HorseRacing.Contracts.Models.Game.Dtos
{
    public class GameDto
    {
        /// <summary>
        /// Код игры
        /// </summary>
        public Guid GameId { get; set; }
        /// <summary>
        /// Статус игры
        /// </summary>
        public StatusType Status { get; set; }
        /// <summary>
        /// Наименование комнаты
        /// </summary>
        public string Name { get; set; } = string.Empty;
        /// <summary>
        /// Дата начала игры
        /// </summary>
        public DateTime DateStart { get; set; }
        /// <summary>
        /// Дата окончания
        /// </summary>
        public DateTime? DateEnd { get; set; }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/dto/game-dto.ts
import type { BaseDto } from '../../../base/dto/base-dto';
import type { GameModeType } from '../enums/game-mode-type-enum';
import type { StatusType } from '../enums/status-type-enum';

export interface GameDto extends BaseDto {
    /** Код игры */
    GameId: string;
    /** Статус игры */
    Status: StatusType;
    /** Режим игры */
    Mode: GameModeType;
    /** Наименование комнаты */
    Name: string;
    /** Дата начала игры */
    DateStart: string;
    /** Дата окончания */
    DateEnd?: string;
}
